src/index.test.js

```diff
@@ -1,5 +1,3 @@
-/* eslint ava/no-only-test:0, ava/no-skip-test:0 */
-import test from 'ava'
```

Comment:
someone should write codemods for this migration or add Ava like assertions to Jest as an optional assertion library. People should be able to write tests in any style they want, as long as it makes them productive.